cranelift-codegen/src/binemit/relaxation.rs

```diff
@@ -178,7 +183,7 @@ fn relax_branch(
     let ctrl_type = dfg.ctrl_typevar(inst);
```

Comment:
Change the comment to 'smallest', not first? (I can't seem to put a review note on the right line for some reason)